### PR TITLE
docs(update): 1st week of communiuty bonding period update provided

### PR DESCRIPTION
**Pull request by soham4abc**
Signed-off-by: Soham Banerjee <sohambanerjee4abc@hotmail.com>

## Project

React UI

### Changes

Weekly update for Community bonding period

### Notes
image

PTAL @shaheemazmalmmd @GMishx

### CODE REVIEW

File: docs/2022/ui/updates/soham/2022-06-02.md [hunk 1]
```diff
@@ -13,20 +13,27 @@ SPDX-FileCopyrightText: 2022 Soham Banerjee <@gmail.com>
...
 <!--truncate-->
 
-### Attendees
+### Discussions
 
-- P1
+- Understood the flow and processes of the weekly updates meetings organised by the Organisation.
+- Discussed and came to fix a suitable time for project specific weekly meetings (Every Tuesday 2:30PM to 3:30PM GMT)
+- Explained the fixes issued during this week and the possiblities of more upcoming fixes.
+- Some project specific doubts cleared regarding the API testing
```
Comment (avinal) [inline docs/2022/ui/updates/soham/2022-06-02.md:28]:
```suggestion
- Understood the flow and processes of the weekly updates meetings organized by the Organisation.
- Discussed and came to fix a suitable time for project-specific weekly meetings (Every Tuesday from 2:30 PM to 3:30 PM GMT)
- Explained the fixes issued during this week and the possibilities of more upcoming fixes.
- Some project-specific doubts cleared regarding the API testing
```

File: docs/2022/ui/updates/soham/2022-06-02.md [hunk 2]
```diff
@@ -13,20 +13,27 @@ SPDX-FileCopyrightText: 2022 Soham Banerjee <@gmail.com>
...
-Different topics discussed.
+- [API Header fix](https://github.com/fossology/FOSSologyUI/pull/210)
+- [Constants exported for browse module](https://github.com/fossology/FOSSologyUI/pull/207)
+- [Updated dependencies of the project](https://github.com/fossology/FOSSologyUI/pull/206)
 
 ### Conclusion and Further Plans
 
-- Conclusions, if any.
+- Currently will be looking for scopes to modularise the larger components into smaller components for better reusablity and flexiblity in the future updates.
+- Will be testing the APIs and if issues found will be reporting them to the mentors and will be issuing respective fixes for them.
```
Comment (avinal) [inline docs/2022/ui/updates/soham/2022-06-02.md:39]:
```suggestion
- Currently will be looking for scopes to modularise the larger components into smaller components for better reusability and flexibility in future updates.
- Will be testing the APIs and if issues are found will be reporting them to the mentors and will be issuing respective fixes for them.
```

**Review by avinal — CHANGES_REQUESTED** [review]
Please consider checking your report using Grammarly, Quillbot, or a similar plugin before opening a PR. 

For signing off add your name instead of your GitHub username. 
i.e 
```bash
    Signed-off-by: FirstName LastName <email@example.com>
```

**Comment by soham4abc** [comment]
> Please consider checking your report using Grammarly, Quillbot, or a similar plugin before opening a PR.
> 
> For signing off add your name instead of your GitHub username. i.e
> 
> ```shell
>     Signed-off-by: FirstName LastName <email@example.com>
> ```

Thanks for the suggestion sir. Will keep them in mind in the future!

**Comment by GMishx** [comment]
@soham4abc I missed it before but the copyright on the file is not correct. Please update it.

**Comment by soham4abc** [comment]
> @soham4abc I missed it before but the copyright on the file is not correct. Please update it.

Copyright on my markdown file... I couldn't get you... Would you please explain... Then I will issue the update asap

**Comment by GMishx** [comment]
There should be correct license and copyright on each file. This one has incorrect email field.

**Pull request merged**